_posts/2022-08-23-aysimab-welcome-test.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "Meet MCT Team C 2022"
+date:   2022-08-23 10:10:42 +0200
+categories: people
+author: Alex Wastnidge, Aysima Karcaaltincaba, Fabian Stordalen, Olve Skjeggedal
+image: /assets/image/2022_08_23_aysimab_teamC.jpeg
+excerpt: "Hello from Team C"
+keywords: jekyll, blog
+---
+
+
+# We are writing here
+<figure style="float: none">
+    <img src='/assets/image/2022_08_23_aysimab_teamC.jpeg' width="auto"/>
+</figure>
+
+## About our team
+
+We can summarise what we discussed our competences are, which diciplines we have etc
+
+<img src='/assets/image/2022_08_25_aysimab_teamC.png' width="auto"/>
```

Comment:
@ajwast feel free to change this file, i just downloaded what you sent and added here but it would be better if you can replace this and maybe rename with original one.

Reply:
there are a couple of issues with this pull request, we will discuss about this tomorrow morning.

Reply:
This is just a draft for now, we will complete it tomorrow morning, would be of course great to discuss it before merging.